<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerWindow</class>
 <widget class="QDialog" name="WorkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Empleado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Clientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Productos</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ta_clientes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ta_productos">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Productos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>360</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>360</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Producto</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_clientdel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_productdel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
